### PR TITLE
fix product item and add nutritions page.

### PR DESCRIPTION
**Pull request by fuatguzel**
- Nutritions line issue,
- Product detail discount and price position issue,
- Category detail product item height problems.

### CODE REVIEW

File: themes/mCommerceTheme/styles/components/flProductItem.json [hunk 1]
```diff
@@ -210,7 +209,7 @@
           "top": 10
         }
       },
-      "paddingTop": 2,
+      "paddingTop": 5,
       "paddingBottom": 2
     },
     "height": 270,
```
Comment (alimoncul) [inline themes/mCommerceTheme/styles/components/flProductItem.json]: activityIndicator not centered inside add to basket button

File: themes/mCommerceTheme/styles/components/LviNutritions.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  ".lviNutritions": {
+    "height": 55
```
Comment (alimoncul) [inline themes/mCommerceTheme/styles/components/LviNutritions.json]: image
too much empty spaces between items. Align separator with text and remove: from header

File: scripts/pages/pgNutritions.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        const processorItems = [];
+        if (this.nutritions == null || Object.keys(this.nutritions).length === 0) {
+            processorItems.push(
+                ListViewItems.getLviEmptyItem({
+                    emptyImage: 'images://empty_nutritions_list.png',
+                    emptyTitle: global.lang.emptyNutritions
+                })
+            );
+        } else {
+            for (var key in this.nutritions) {
```
Comment (alimoncul) [inline scripts/pages/pgNutritions.ts]: use map or forEach

File: scripts/pages/pgNutritions.ts [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    }
+    public onLoad() {
+        super.onLoad?.();
+        this.getNutritions();
+        this.initListView();
+        this.headerBar.title = global.lang.nutritions;
+    }
+    getNutritions() {
+        this.nutritions = this.route.getState().routeData.product?.nutritions;
+        console.info('this.nutritions', this.nutritions);
```
Comment (alimoncul) [inline scripts/pages/pgNutritions.ts]: remove logs

File: .ui/library/flProductDetailButtonPriceSection.cpx [hunk 5]
```diff
@@ -166,37 +166,66 @@
 			}
```
Comment (alimoncul) [inline .ui/library/flProductDetailButtonPriceSection.cpx]: use a flexLayout component for application titlelayout on 5 tab pages

File: .ui/library/lviNutritions.cpx [hunk 6]
```diff
@@ -0,0 +1,41 @@
+{
+	"components": [
```
Comment (alimoncul) [inline .ui/library/lviNutritions.cpx:2]: generate appIcon and use not transparent image for titlelayotu

**Review by alimoncul — CHANGES_REQUESTED** [review]

File: themes/mCommerceTheme/styles/components/flProductItem.json (hunk 1 above)
Comment (fuatguzel) [reply themes/mCommerceTheme/styles/components/flProductItem.json]: DONE

File: themes/mCommerceTheme/styles/components/LviNutritions.json (hunk 2 above)
Comment (fuatguzel) [reply themes/mCommerceTheme/styles/components/LviNutritions.json]: DONE

File: scripts/pages/pgNutritions.ts (hunk 3 above)
Comment (fuatguzel) [reply scripts/pages/pgNutritions.ts]: DONE.

File: scripts/pages/pgNutritions.ts (hunk 4 above)
Comment (fuatguzel) [reply scripts/pages/pgNutritions.ts]: DONE

File: .ui/library/flProductDetailButtonPriceSection.cpx (hunk 5 above)
Comment (fuatguzel) [reply .ui/library/flProductDetailButtonPriceSection.cpx]: Miss

File: .ui/library/lviNutritions.cpx (hunk 6 above)
Comment (fuatguzel) [reply .ui/library/lviNutritions.cpx:2]: Miss

File: scripts/pages/pgAddReview.ts [hunk 7]
```diff
@@ -7,13 +7,18 @@ import { getProductImageUrl, postProductReview } from 'service/commerce';
...
 
 export default class PgAddReview extends withDismissAndBackButton(PgAddReviewDesign) {
     product: Product;
     constructor(private router?: Router, private route?: Route) {
         super({});
         this.btnSendReview.on(Button.Events.Press, () => {
-            this.postReview(this.product._id, this.flReviewAndRateProduct.rate, this.flReviewAndRateProduct.comment);
+            if (this.flReviewAndRateProduct.rate !== 0) {
+                store.dispatch(storeActions.AddNewRate({ isRateAdded: true }));
+                this.postReview(this.product._id, this.flReviewAndRateProduct.rate, this.flReviewAndRateProduct.comment);
```
Comment (alimoncul) [inline scripts/pages/pgAddReview.ts]: make dispatch on success of postReview

**Pull request merged**